1.8/usage/tutorials/jenkins/index.md

```diff
@@ -66,7 +66,8 @@ as NFS, CIFS, and so on.
 
 If you only want to run Jenkins in a development environment, it's trivial
 to pin it to a single agent in the DC/OS cluster. Create the file
-`options.json` with the configuration below:
+`options.json` with the configuration below, modifying `pinned-hostname` to
+correspond with an agent IP in your DC/OS cluster:
 
 ```bash
 $ cat options.json
```

Comment:
We should update this line to `storage.pinned-hostname` since the package has changed.